### ISSUE TITLE
App does not allow voting with reserved balance

### ISSUE THREAD

**Issue by Ank4n** [polkadot-js/apps#10837]
With the changes in pool to allow open gov participation, the staked balance will migrate from locked (frozen) to reserved (held). The conviction-voting pallet allows [total balance](https://github.com/paritytech/polkadot-sdk/blob/master/substrate/frame/conviction-voting/src/lib.rs#L397) including reserved balance to vote.

But the UI does not allow to vote with reserved balance.
![image](https://github.com/user-attachments/assets/02b8ee8d-7f78-4e1f-afa5-5aa4dff1eab5)

![image](https://github.com/user-attachments/assets/a2ce8376-82fb-4003-9a56-1c35ea82759b)

The above scenario works if I call the extrinsic directly.
![image](https://github.com/user-attachments/assets/37163e57-9edb-49bf-ac9f-2d5331e8cac0)
https://westend.subscan.io/extrinsic/22143408-2

This bug is probably due to the old democracy pallet that only allowed locked (frozen) funds for vote.

**Comment by TarikGul** [comment]
@Ank4n Is there anything specific in the runtime that would say this logic is valid? Like a call, storage entry, etc.?

Basically I want to do a check like: 'If this is available on chain then voting balance should include reserves'

**Comment by Ank4n** [comment]
> @Ank4n Is there anything specific in the runtime that would say this logic is valid? Like a call, storage entry, etc.?
> 
> Basically I want to do a check like: 'If this is available on chain then voting balance should include reserves'

@TarikGul If `pallet-conviction-voting` is used for voting, you can use reserved balance.

**Comment by TarikGul** [comment]
@Ank4n So the pallet just needs to be active in the runtime?

**Comment by Ank4n** [comment]
> @Ank4n So the pallet just needs to be active in the runtime?

Yeah the above pallet supports reserved balance since [this commit](https://github.com/paritytech/polkadot-sdk/commit/5d81f23f8f8734c5d5fbb845e06ef972e952c339) which was well over a year ago. I guess it is okay to assume if this pallet is used, that it supports reserved balance? (not sure if any chain is using the soft deprecated `pallet-democracy` which does not allow reserved balance for voting).

**Comment by Ank4n** [comment]
How does the voting UI work? I am guessing it is hardcoded to support only these two pallets?
